.github/workflows/ci-cadence-integ-test-disable-sticky.yml

```diff
@@ -0,0 +1,26 @@
+name: Cadence Integration Test
+on:
+  pull_request:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  tests:
+    name: "Integration testing"
```

Comment:
Let's add `with stickyCache disabled` otherwise it's hard to differentiate with the other one 😂

Reply:
Done